org.blender.Blender.json

```diff
@@ -116,7 +116,8 @@
                 "--enable-decoder=mp3",
                 "--enable-decoder=mpeg4",
                 "--enable-decoder=pcm_s16le",
-                "--enable-decoder=theora"
+                "--enable-decoder=theora",
+                "--disable-inline-asm"
```

Comment:
use the [patch](https://github.com/flathub/io.qt.qtwebengine.BaseApp/blob/branch/5.15-23.08/patches/chromium-ffmpeg-binutils.patch)